blockchains/ethereum/assets/0xfF759254e578bc7F725428CEcb612A502E604f42/info.json

```
@@ -3,9 +3,9 @@
     "symbol": "BATG",
     "type": "ERC20",
     "decimals": 0,
-    "description": "-",
-    "website": "",
+    "description": "Utility Token",
+    "website": "Various Air Drop sites",
```

```suggestion
    "website": "Various Air Drop sites",
```

wrong format of the website, don't include if there is no website